drivers/builtin/src/nist_kw.c

```diff
@@ -51,6 +51,26 @@ static void calc_a_xor_t(unsigned char A[KW_SEMIBLOCK_LENGTH], uint64_t t)
     }
 }
 
+static int verify_input(psa_key_attributes_t *attributes, mbedtls_svc_key_id_t *key){
```

Comment:
Please make `attributes` a local variable (and don't forget to clean up on error paths), since it isn't used outside this function.